app/MMB/src/scenes/timeline/Events.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import * as React from 'react';
+import { DateFormatter, DateUtils } from '@kiwicom/mobile-localization';
+import { View } from 'react-native';
+
+import Event, { type Props as EventType } from './event/Event';
+import Day from './event/Day';
+
+type Props = {|
+  events: Array<EventType>,
+|};
+
+export default function Events(props: Props) {
+  const { events } = props;
+  return (
+    <View>
+      {events.map((event, index) => {
+        const dayLabel =
+          DateUtils.diffInDays(event.dateTime, new Date()) === 0
+            ? 'today'
+            : DateFormatter(event.dateTime).formatToDate();
+        return (
+          <View key={event.description + index}>
+            {event.first && <Day label={dayLabel} />}
+            <Event
+              icon={event.icon}
+              description={event.description}
+              comment={event.comment}
+              dateTime={event.dateTime}
+              location={event.location}
+              first={event.first}
+            />
```

Comment:
Maybe you could pass the event object instead.
`<Event event={event} /> `

Reply:
> Maybe you could pass the event object instead.

It does not sound like a good idea.